src/render_tasks/d3d12_equirect_to_cubemap.hpp

```diff
@@ -112,8 +112,8 @@ namespace wr
 			//Get allocator from pool
 			auto allocator = static_cast<D3D12TexturePool*>(src_texture.m_pool)->GetMipmappingAllocator();
 
-			d3d12::TextureResource* cubemap_text = static_cast<d3d12::TextureResource*>(src_texture.m_pool->GetTexture(src_texture.m_id));
-			d3d12::TextureResource* envmap_text = static_cast<d3d12::TextureResource*>(dst_texture.m_pool->GetTexture(dst_texture.m_id));
+			d3d12::TextureResource* cubemap_text = static_cast<d3d12::TextureResource*>(src_texture.m_pool->GetTextureResource(src_texture));
```

Comment:
This seems to happen more with your naming, the "_text" suffix is slightly confusing for me. This is a minor thing though